app/config/publications/overview-table-columns.js

```diff
@@ -1,4 +1,25 @@
 export default [
+  {
+    keyName: 'speedProcedure',
+    translationKey: 'publications-table-speed-procedure',
+    translationKeySmall: 'publications-table-speed-procedure-small',
```

Comment:
Deze keys hebben nog hun oude namen. Het is wat verwarrend, maar dan blijft de configuratie in localStorage correct.

Reply:
Aangezien publicaties nog niet in productie gebruikt worden, zou ik me niet aantrekken om hier een breaking change te doen indien nodig.